web/app/scripts/app.js

```diff
@@ -31,6 +31,30 @@
     }
 
     /* ngInject */
+    function TranslationConfig($translateProvider, $windowProvider) {
+        // Configure the location of the translation json files on the server
+        $translateProvider.useStaticFilesLoader({
+            prefix: 'i18n/',
+            suffix: '.json'
+        });
+
+        // Log untranslated tokens to console
+        $translateProvider.useMissingTranslationHandlerLog();
+
+        // Enable sanitization of token
+        $translateProvider.useSanitizeValueStrategy('sanitizeParameters');
+
+        // Get the browser's navigator object and use for determining preferred language
+        var navigator = $windowProvider.$get().navigator;
+        var language = (navigator.languages ? navigator.languages[0] :
+                        (navigator.language || navigator.userLanguage)).substring(0,2);
+
+        // Interpret Zulu ('zu') to request dev language ('exclaim')
+        $translateProvider.preferredLanguage(language === 'zu' ? 'exclaim' : language);
```

Comment:
That's a really good point I hadn't considered. I'll take a look through the supported languages in Chrome, and will switch to a lesser used language.